www_src/components/option-panel/option-panel.jsx

```diff
@@ -0,0 +1,82 @@
+var React = require('react/addons');
+
+// Internal -- just the container
+var OptionPanel = React.createClass({
```

Comment:
it might be worth doing these components as one component per file and making this a plain js `module.exports = { require("checkbox"), ..., ... }`